packages/application-shell/src/utils/get-correlation-id/get-correlation-id.ts

```diff
@@ -1,4 +1,6 @@
-import uuid from 'uuid/v4';
+// ESLint is not able to follow this correct import.
+// eslint-disable-next-line import/named
+import { v4 as uuid } from 'uuid';
```

Comment:
This is a bit odd. The typings indicate that the usage is correct. ESlint still warns

<img width="437" alt="CleanShot 2020-03-02 at 10 30 05@2x" src="https://user-images.githubusercontent.com/1877073/75663572-55470900-5c71-11ea-89ba-f8129d3dcfce.png">

I think due to the structure of the module it can't follow the import

<img width="498" alt="CleanShot 2020-03-02 at 10 34 29@2x" src="https://user-images.githubusercontent.com/1877073/75663612-6859d900-5c71-11ea-97b2-c33c235df9e4.png">

cause uuid then defines exports like this in uuid's built files

```js
export { default as v1 } from './v1.js';
export { default as v3 } from './v3.js';
export { default as v4 } from './v4.js';
export { default as v5 } from './v5.js';
```

I logged things on the usages of the module

<img width="644" alt="CleanShot 2020-03-02 at 10 31 29@2x" src="https://user-images.githubusercontent.com/1877073/75663820-c090db00-5c71-11ea-9c26-c1739feb4be5.png">

...and things work

Maybe just another opinion of @emmenko or @adnasa would help.

Reply:
Hmm odd, I can check out the branch and take a look if you want

Reply:
Yeah that'd be nice. It's not urgent I think. There are about 5-6 versions of uuid flying around but it would be nice to know if it was truely ESLint failing here.

Reply:
The problem is that the hoisted version is `3.4.0`, as there are many different versions installed. So eslint is correct, because it findd the older version.

<img width="734" alt="image" src="https://user-images.githubusercontent.com/1110551/75680291-94855200-5c91-11ea-8874-b93cff98a1e6.png">

Reply:
Probably better to wait with the update, as using deps resolutions might break other libraries.

Reply:
Yes, I saw that too. However. E.g. the sdk has its v7 resolved. On the MC I already managed to update it. I will look into it later but am also fine with skipping.